src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -0,0 +1,156 @@
+import { css, CSSResultGroup, html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import {
+  StatisticsValidationResult,
+  validateStatistics,
+} from "../../../data/history";
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import "@material/mwc-button/mwc-button";
+import "../../../components/data-table/ha-data-table";
+import type { DataTableColumnContainer } from "../../../components/data-table/ha-data-table";
+import { showAlertDialog } from "../../../dialogs/generic/show-dialog-box";
+import { showFixStatisticsUnitsChangedDialog } from "./show-dialog-statistics-fix-units-changed";
+
+@customElement("developer-tools-statistics")
+class HaPanelDevStatistics extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @state() private _data: (StatisticsValidationResult["data"] & {
+    type: string;
+  })[] = [];
+
+  protected firstUpdated() {
+    this._validateStatistics();
+  }
+
+  private _columns: DataTableColumnContainer = {
+    statistic_id: {
```

Comment:
Would be nice if we detect that it's an entity ID that people can open the more info dialog maybe?

Reply:
bonus points if we could open a dialog with a graph with all the available stats data

Reply:
oh a column with oldest data point maybe?

Reply:
(ok, future PRs maybe)